src/main/java/me/totalfreedom/totalfreedommod/command/Command_tag.java

```diff
@@ -18,7 +18,7 @@
 
     public static final List<String> FORBIDDEN_WORDS = Arrays.asList(new String[]
     {
-        "admin", "owner", "moderator", "developer", "console"
+        "admin", "owner", "moderator", "developer", "console", "sra", "sta", "sa", "Super Admin", "Telnet Admin", "Telnet Clan Admin", "Senior Admin"
```

Comment:
Super Admin, Telnet Admin etc. need to be lowercase because the tag command will make the tag all lowercase to check for forbidden words. ``final String rawTag = ChatColor.stripColor(outputTag).toLowerCase();``